tensorflow/lite/micro/kernels/xtensa/xtensa_conv.h

```diff
@@ -50,12 +60,10 @@ TfLiteStatus ConvEvalHifi16(TfLiteContext* context, TfLiteNode* node,
                             const TfLiteEvalTensor* filter,
                             const TfLiteEvalTensor* bias,
                             TfLiteEvalTensor* output);
+#else
+  TfLiteStatus ConvReferenceEvalInt16(TfLiteContext* context, TfLiteNode* node);
 #endif  // defined (HIFI4_INTERNAL)
 
-TfLiteStatus ConvReferenceEvalInt8(TfLiteContext* context, TfLiteNode* node);
```

Comment:
Does it help if we keep ConvReferenceEvalInt8() and ConvReferenceEvalInt16() outside the #if define's? It does not affect code size if they're not used, and it reduces burden of future maintenance if we do not have to worry about which #if defined() condition/branch they belong to.

Reply:
Done.